apps/coscrad-frontend/src/components/tenant/tenant-logo.tsx

```diff
@@ -1,12 +1,15 @@
 import { useContext } from 'react';
 import { ConfigurableContentContext } from '../../configurable-front-matter/configurable-content-provider';
+import { StyledImageInUILayout } from '../../utils/generic-components/presenters/styled-image-in-ui-layout';
 
 export const TenantLogo = (): JSX.Element => {
     const { organizationLogoUrl } = useContext(ConfigurableContentContext);
 
     return (
-        <div className="organization-logo">
-            <img src={organizationLogoUrl} alt="organization logo" />
-        </div>
+        <StyledImageInUILayout
+            sx={{ width: { xs: '40px', sm: '55px', md: '61px' } }}
```

Comment:
Is there occasion to pass different values for different "styled image in UI layout"s? If not, why not build an abstraction around this and set these values for once and for all?

Reply:
These `sx` properties will vary widely on the use case.  `sx` is not particular to the component I created.  When you use `styled` MUI automatically allows you to use the `sx` prop on the component you're styling.